src/common/components/htmlToReact/utils/validateTrustedOriginsFormat.ts

```diff
@@ -0,0 +1,12 @@
+export default function validateTrustedOriginsFormat(
+  trustedOrigins?: string[],
+) {
+  if (
+    trustedOrigins?.length &&
+    !trustedOrigins.every((origin) => new URL(origin).origin !== 'null')
```

Comment:
Shouldn't it? -- Then it would be misconfigured and the configuration should be changed.

Reply:
- Adding a comment about [URL constructor raising TypeError](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL#exceptions) here could make it more readily apparent what the possible errors raised here are IMHO.

- There's also Array.prototype.[some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) which could be used in place of !every.

- The `trustedOrigins?.length` check is unnecessary because
  > some() acts like the "there exists" quantifier in mathematics. In particular, for an empty array, it returns false for 
      any condition.

   and:

  > every acts like the "for all" quantifier in mathematics. In particular, for an empty array, it returns true.

Reply:
The TypeError is not thrown strictly enough:
```
new URL("asdfa://niko.").origin
'null'
new URL("asdfa://niko.")
URL {origin: 'null', protocol: 'asdfa:', username: '', password: '', host: '', …}
```

Reply:
I added a comment about raising the custom error.